local/network.go

```diff
@@ -106,8 +104,6 @@ type localNetwork struct {
 }
 
 var (
-	//go:embed default
-	embeddedDefaultNetworkConfigDir embed.FS
```

Comment:
would like for the genesis to be beign loaded here, have all the genesis changes visible at this level, and only have a function in utils to get the new cChainConfig field contents